### DISCUSSION TITLE
Is this still "A Serverless v1.x plugin"?

### DISCUSSION THREAD

**Discussion by johnmee** [UnitedIncome/serverless-python-requirements#665] [Q&A]
The opening sentence states that this is a "Serverless v1.x plugin".

Serverless v1 is deprecated and, today, we're up to v2.76.0.
Does this plugin do anything for v2?

**Comment by pgrzesik** [comment]
Hello @johnmee - thanks for pointing that out - plugin is compatible with `v2` as well as with upcoming `v3` release. The `v1` version of the Framework is deprecated and plugin might no longer support it (based on `peerDependencies`, the lowest supported version is `v2.32`.
